changelog/3693.misc.rst

```diff
@@ -0,0 +1 @@
+Provide Zstandard support through the ``backports.zstd`` library for Python before 3.14.
```

Comment:
Let's explicitly state that the `zstd` extra now installs `backports.zstd` package instead of `zstandard`